translator/reader/reader.cpp

```diff
@@ -165,6 +165,10 @@ void UsdArnoldReader::readStage(UsdStageRefPtr stage, const std::string &path)
 {
     // set the stage while we're reading
     _stage = stage;
+    if (stage == nullptr) {
+        AiMsgError("[usd] Impossible to read USD Stage %s", _filename.c_str());
```

Comment:
I don't think `Impossible to read USD Stage` is the best phrasing here. We are not reading an USD stage, but we are opening a stage from from a file that USD happens to support. Maybe, `Unable to open %s` or `Unable to create USD stage from %s`